js/build.js

```diff
@@ -63,6 +63,18 @@ Fliplet().then(function() {
             }, 1000);
           }
         },
+        createUserProfile(entry) {
+          entry = entry || {};
+          if (!entry.dataSourceId || !entry.id) {
+            return;
+          }
+
+          return {
```

Comment:
is this going to be a standard to look for inside `Fliplet.Profile.get('user')` ?

Reply:
Yes. It won't contain the user details such as email and name, but it'll include the unique identifier for the particular user login.

Reply:
A type (in this case: `dataSource`) and any addition details that help uniquely identify the user.

Other types might be `saml2`, `oauth2`, `fliplet` etc.